spec/request/appeals_request_spec.rb

```diff
@@ -40,8 +40,10 @@
     end
 
     context 'with a not authorized response' do
-      it 'returns a 502' do
+      it 'returns a 502 and logs an error level message' do
```

Comment:
Minor, but, it's good to split up specs to a single responsibility (within reason).  e.g.
```ruby
not_authorized = { cassette_name: 'appeals/not_authorized' }
context  'with a not authorized response', vcr: not_authorized do
  it 'returns 502'
  it 'logs an error level message'
end